ext/mysqli/mysqli.stub.php

```diff
@@ -547,225 +547,225 @@ function mysqli_connect_errno(): int {}
+function mysqli_stmt_init(mysqli $connection): mysqli_stmt|false {}
 
-function mysqli_stmt_insert_id(mysqli_stmt $mysql_stmt): int|string {}
+function mysqli_stmt_insert_id(mysqli_stmt $stmt): int|string {}
 
 #if defined(MYSQLI_USE_MYSQLND)
-function mysqli_stmt_more_results(mysqli_stmt $mysql_stmt): bool {}
+function mysqli_stmt_more_results(mysqli_stmt $stmt): bool {}
 
-function mysqli_stmt_next_result(mysqli_stmt $mysql_stmt): bool {}
+function mysqli_stmt_next_result(mysqli_stmt $stmt): bool {}
 #endif
 
-function mysqli_stmt_num_rows(mysqli_stmt $mysql_stmt): int|string {}
+function mysqli_stmt_num_rows(mysqli_stmt $stmt): int|string {}
 
-function mysqli_stmt_param_count(mysqli_stmt $mysql_stmt): int {}
+function mysqli_stmt_param_count(mysqli_stmt $stmt): int {}
 
-function mysqli_stmt_prepare(mysqli_stmt $mysql_stmt, string $query): bool {}
+function mysqli_stmt_prepare(mysqli_stmt $stmt, string $query): bool {}
 
-function mysqli_stmt_reset(mysqli_stmt $mysql_stmt): bool {}
+function mysqli_stmt_reset(mysqli_stmt $stmt): bool {}
 
-function mysqli_stmt_result_metadata(mysqli_stmt $mysql_stmt): mysqli_result|false {}
+function mysqli_stmt_result_metadata(mysqli_stmt $stmt): mysqli_result|false {}
 
-function mysqli_stmt_send_long_data(mysqli_stmt $mysql_stmt, int $param_nr, string $data): bool {}
+function mysqli_stmt_send_long_data(mysqli_stmt $stmt, int $index, string $data): bool {}
 
-function mysqli_stmt_store_result(mysqli_stmt $mysql_stmt): bool {}
+function mysqli_stmt_store_result(mysqli_stmt $stmt): bool {}
 
-function mysqli_stmt_sqlstate(mysqli_stmt $mysql_stmt): ?string {}
+function mysqli_stmt_sqlstate(mysqli_stmt $stmt): ?string {}
 
-function mysqli_sqlstate(mysqli $mysqli_link): ?string {}
+function mysqli_sqlstate(mysqli $connection): ?string {}
 
 function mysqli_ssl_set(
-    mysqli $mysql_link,
+    mysqli $connection,
     string $key,
-    string $cert,
+    string $certificate,
     string $certificate_authority,
     string $certificate_authority_path,
-    string $cipher
+    string $cipher_algorithms
 ): bool {}
 
-function mysqli_stat(mysqli $mysql_link): string|false {}
+function mysqli_stat(mysqli $connection): string|false {}
 
-function mysqli_store_result(mysqli $mysql_link, int $flags = 0): mysqli_result|false {}
+function mysqli_store_result(mysqli $connection, int $flags = 0): mysqli_result|false {}
 
-function mysqli_thread_id(mysqli $mysql_link): int {}
+function mysqli_thread_id(mysqli $connection): int {}
 
 function mysqli_thread_safe(): bool {}
 
-function mysqli_use_result(mysqli $mysql_link): mysqli_result|false {}
+function mysqli_use_result(mysqli $connection): mysqli_result|false {}
 
-function mysqli_warning_count(mysqli $mysql_link): int {}
+function mysqli_warning_count(mysqli $connection): int {}
 
-function mysqli_refresh(mysqli $mysqli_link, int $options): bool {}
+function mysqli_refresh(mysqli $connection, int $options): bool {}
```

Comment:
The `$options` parameter appears to be a bitmask. These are called `$flags` elsewhere. Doing the same here would be more consistent.

https://www.php.net/manual/en/mysqli.refresh.php